_author_instructions/01_policies.md

```diff
@@ -204,9 +204,17 @@ After this time, a revised manuscript may be handled as a new submission.
 
-## After acceptance
+## Upon acceptance
+After acceptance of your article, the first task is to prepare an appropriate "ASAP" version of the article which will be posted to the journal's website prior to publication of the next issue. Several steps, outlined below, are required. Note that ASAP version will essentially be final, as only minor corrections as detailed below will be permitted prior to publication as part of an issue.
+
+- Make sure the metadata is complete in the submission (including institution, ORCID, abstract). 
+- Have the managing editor assign a DOI to the article (correct metadata is needed for this).
+- Do a final check for issues with grammar, word choice, spelling, formatting (including issues with the LaTeX template, placement of figures, etc.) and typos, as normally would be done at the proofs stage. Please [reference the appropriate part of the author instructions](https://livecomsjournal.github.io/authors/policies/#writing-style-and-editing) for what to look for. Note that authors bear ultimate responsibility for ensuring the writing is of good quality. Editors may suggest improvements, but this is at their discretion.
+- Enable the LiveCoMS footer by including the `pubversion` class option in the document preamble, fill in the `\datereceived`, `\dateaccepted`, and `\pubDOI` fields, which the managing editor will provide after the Metadata is verified to be correct. You should make the PDF accessible on the GitHub repository as `release/LiveCoMS_Article_VX.pdf`, where 'X' is the current article verrsion. The related header image should be at least 1200 pixels wide and in `.jpg` format, and should be available on the GitHub repository as `release/header_VX.jpg`, where again 'X' is the current article version, and notify the editor when this has been done.
+- Provide the editor who managed the article a "lede", a 30 word summary of the article that will accompany the article in social media posts.
+- After being published as an ASAP article, you will have a final chance to fix minor typos and inaccuracies (but not substantive content) before the article is published in the next regularly scheduled issue.  Beyond that, any typos will remain in the LiveCOMS version until the next peer-reviewed version.  Changes can of course be made at any time in the authors’ maintained version on GitHub, which is linked from the article itself.  
+- When assigned to a journal issue, you will be notified by the editors about the volume, year, and article number, which should be used to set the `\pubvolume`, `\pubyear`, and `\articlenum` fields. Then recompile the PDF, and check it into your GitHub repository under the folder `release/LiveCoMS_Article_VX.pdf`, where 'X' is the current article verrsion, along with any final corrections as noted above.
```

Comment:
Seems reasonable. Should they also be doing a formal "release" on their GitHub repo at the same time?

Reply:
I guess I thought it was implied that THAT would be the release version, but maybe it's not specific enough. What were you thinking might work better?

Reply:
I'm just saying that GitHub has a "releases" tab where you can make formal releases of your code. I'm suggesting that maybe when they do this, they also make a formal release on GitHub under that tab. Or are we talking past each other and that's what you mean? This was not clear in your instructions; it sounds like you're just saying that the release is just a separate copy of the PDF in their repo, but they don't do anything particular on the GitHub repo other than just place the PDF there.

Reply:
Ah, yes, that makes sense.  I will link: https://help.github.com/articles/creating-releases/ and say they should create as a release.  Then we can retrieve the pdf there.